pkg/sql/logictest/testdata/logic_test/drop_database

```diff
@@ -219,19 +226,20 @@ CREATE TABLE constraint_db.t2 (
 statement ok
 DROP DATABASE constraint_db CASCADE
 
-query TTT
+query TT
```

Comment:
how does this help us? We don't create this many jobs, do we? Should we just `skipif` this query?

Reply:
What do you think of this? Why are there two jobs in the new schema changer? It seems to me like this query is cheating because there are other jobs which meet these more lax criteria